setup.cfg

```diff
@@ -1,3 +1,90 @@
+
+[metadata]
+name             = pymodbus
+version          = attr: pymodbus.__version__
+author           = attr: pymodbus.__author__
+maintainer       = attr: pymodbus.__maintainer__
+license          = 'BSD-3-Clause'
+platforms        = 'Linux', 'Mac OS X', 'Win'
+description      = "A fully featured modbus protocol stack in python"
+keywords         = 'modbus, asyncio, scada, client, server, simulator'
+long_description = file: README.rst, CHANGELOG.rst, LICENSE
+url              = 'https://github.com/riptideio/pymodbus/'
+classifiers      = 
+    Development Status :: 4 - Beta
+    Programming Language :: Python :: 3.8 :: 3.9 :: 3.10
+    Environment :: Console
+    Framework :: Twisted :: Tornado :: Asyncio
+    Intended Audience :: Developers :: Modbus testers
+    License :: OSI Approved :: BSD License
```

Comment:
I am not so deep in that Licence thing....do we have a BSD Licence? What kind of Licence is this one https://github.com/riptideio/pymodbus/blob/dev/LICENSE ? @dhoomakethu

Reply:
Just saw, that this is the old state as well...